src/main.rs

```diff
@@ -38,7 +39,10 @@ extern crate term_size;
 
 use chrono::prelude::*;
 use clap::{App, AppSettings, Arg, ArgMatches, SubCommand};
+#[cfg(unix)]
 use libc::pid_t;
+#[cfg(windows)]
+type pid_t = u32;
```

Comment:
are you sure this is the right size?

Reply:
I *think* it’s the right size.  Inside the win32 api, pid’s are a DWORD type - which are unsigned long’s (which are 32 bits on windows): https://github.com/retep998/winapi-rs/blob/0.3/src/shared/minwindef.rs#L21 .